### PR TITLE
acknowledgements for pre-open-source committers

### PR DESCRIPTION
**Pull request by lanther**
Adding acknowledgements - this is based on commits to Twist/Torq repos, so only includes contributions in terms of code/documentation etc (unfortunately there's no way to quantify contributions in terms of helping others, etc).

Please add a comment on the line with your name if you want me to add a link to your public github handle.

### CODE REVIEW

File: ACKNOWLEDGEMENTS.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+Atishay Jain
+Bartosz Borowski
+Ben Helleman
+Bin Liu
+Botezatu Mihai Catalin ([@botezatumihaicatalin](https://github.com/botezatumihaicatalin))
+Brad Bambara
+Brenda Burden
+Channey Jungmin Park
+Charan Raj
+Charles Piña
```
Comment (charlespina) [inline ACKNOWLEDGEMENTS.md]: charlespina

File: ACKNOWLEDGEMENTS.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+Chris Millar
+Christian Lista Nicoloso
+Clyde Sy
+Conor Buckley
+Corey Birnbaum
+Cristian Cibu
+David Pond
+David Xu ([davidyxu](https://github.com/davidyxu))
+Devavrat Tomar
+Dinesh Yalagandula
```
Comment (awkawk) [inline ACKNOWLEDGEMENTS.md]: Dinesh = @yalagand

File: ACKNOWLEDGEMENTS.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+Mark Rausch
+Mark Shepherd ([@markshepherd](https://github.com/markshepherd))
+Matt Joss
+Maxime Tchouanmou
+Mayank Jain
+Michael Jordan ([@majornista](https://github.com/majornista))
+Michael Mortimer
+Michael Smith ([@lanther](https://github.com/lanther))
+Michael Taylor ([@cuberoot](https://github.com/cuberoot))
+Mike Yang
```
Comment (huacyang) [inline ACKNOWLEDGEMENTS.md]: ([@huacyang](https://github.com/huacyang))

File: ACKNOWLEDGEMENTS.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+Christian Lista Nicoloso
+Clyde Sy
+Conor Buckley
+Corey Birnbaum
+Cristian Cibu
+David Pond
+David Xu ([davidyxu](https://github.com/davidyxu))
+Devavrat Tomar
+Dinesh Yalagandula
+Elcin Erkin
```
Comment (elcinerkin) [inline ACKNOWLEDGEMENTS.md]: ([@elcinerkin](https://github.com/elcinerkin))

File: ACKNOWLEDGEMENTS.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+Nathan Pham
+Nathan Plesnicher
+Oscar Bolaños
+Piotr Walczyszyn
+Rahul Jain
+Ram Narain Yadav
+Rick Seeler
+Robert Taylor
+Rohit Suri
+Sanket Radder
```
Comment (sanketra) [inline ACKNOWLEDGEMENTS.md]: [@sanketra](https://github.com/sanketra)

File: ACKNOWLEDGEMENTS.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+Anirudh Sasikumar ([@anirudhsasikumar](https://github.com/anirudhsasikumar))
+Ankit Kumar Sinha
+Apoorva Shetty
+Archie Lim
+Ari Fuchs ([@afuchs](https://github.com/afuchs))
+Armin Roșu
+Arnon Ziklik
+Ashish Gupta
+Ashutosh Sharma ([@toshsharma](https://github.com/toshsharma))
+Atishay Jain
```
Comment (atishay) [inline ACKNOWLEDGEMENTS.md]: @atishay

File: ACKNOWLEDGEMENTS.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+Kaori Mikawa
+Karim Nahas
+Kavit Shah
+Keith Mashinter
+Kenny Meyers
+Kerri Shotts ([@kerrishotts](https://github.com/kerrishotts))
+Kirk Mattson
+Leena Tahilramani
+Li Sun
+Lily Khong
```
Comment (lkhong) [inline ACKNOWLEDGEMENTS.md]: @lkhong

File: ACKNOWLEDGEMENTS.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+Ben Helleman
+Bin Liu
+Botezatu Mihai Catalin ([@botezatumihaicatalin](https://github.com/botezatumihaicatalin))
+Brad Bambara
+Brenda Burden
+Channey Jungmin Park
+Charan Raj
+Charles Piña ([@charlespina](https://github.com/charlespina))
+Chris Millar
+Christian Lista Nicoloso
```
Comment (chnicoloso) [inline ACKNOWLEDGEMENTS.md]: @chnicoloso Thanks Michael!

File: ACKNOWLEDGEMENTS.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+Twist, and the libraries around Twist, evolved over a number of years at Adobe, and many people have contributed to it during this time. Twist and its predecessors are used in one form or another, in a quite a few Adobe products! Unfortunately, for legal reasons, we've had to start with clean repos when open-sourcing Twist and React-Twist, which means that there's no record of these contributions in the commit history.
+
+We'd like to thank the following people, in addition to the contributors in Github, who've directly contributed to Twist and its predecessors. These contributions range from features, bug fixes, documentation, and tests, among others. This also includes contributors to libraries that aren't yet open sourced as part of Twist - we're gradually going to be open sourcing more libraries that we think are useful to people outside Adobe, so watch this space :)
+
+Thanks, everyone!
+
+## Pre-Open-Source Contributors to Twist and Pre-Twist (aka Torq) Libraries
+
+Aaron Brownlee ([@misterbrownlee](https://github.com/misterbrownlee))
+Abhinav Gupta
```
Comment (abhigupta1702) [inline ACKNOWLEDGEMENTS.md]: @abhigupta1702

File: ACKNOWLEDGEMENTS.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+Abhishek Singh
+Adrian Brojbeanu ([@theadriann](https://github.com/theadriann))
+Alan Stearns ([@astearns](https://github.com/astearns))
+Alessandro Crugnola
+Alex Dixon
+Alex Normand
```
Comment (alexnormand) [inline ACKNOWLEDGEMENTS.md]: @alexnormand

File: ACKNOWLEDGEMENTS.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+Srinivas Annam ([@sannam](https://github.com/sannam))
+Srishti Gupta
+Stuart Knightley
+Subbi Pillai
+Sujai S
+Taylor Millett
+Terry Ridgway
+Tony Constantinides
+Trevor McCauley
+Vijay Jain
```
Comment (xitter) [inline ACKNOWLEDGEMENTS.md]: ([@xitter](https://github.com/xitter))

Comment (xitter) [inline ACKNOWLEDGEMENTS.md]: [@xitter](https://github.com/xitter)

File: ACKNOWLEDGEMENTS.md [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+Jeremy Mitchell
+Jin Pae
+Joe Winifred Poobalarayen
+Joel Richard
+John Brinkman
+Jon Kuperman ([@jkup](https://github.com/jkup))
+Jonathan Brown
+Jürgen Toth
+Kaho Lo ([@kahol](https://github.com/kahol))
+Kanuj Kumar
```
Comment (kanujkumar) [inline ACKNOWLEDGEMENTS.md]: @kanujkumar(https://github.com/kanujkumar)

File: ACKNOWLEDGEMENTS.md [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+Nathan Plesnicher
+Oscar Bolaños
+Piotr Walczyszyn
+Rahul Jain
+Ram Narain Yadav
+Rick Seeler
+Robert Taylor
+Rohit Suri
+Sanket Radder ([@sanketra](https://github.com/sanketra))
+Scott Morrice
```
Comment (malinkie) [inline ACKNOWLEDGEMENTS.md]: `([@malinkie](https://github.com/malinkie))`

File: ACKNOWLEDGEMENTS.md [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+Henry Fang
+Henry Ping
+Hongyi Zhang
+Ian Wehrman ([@iwehrman](https://github.com/iwehrman))
+Ion Rosca ([@rosca](https://github.com/rosca))
+James Thompson
+Jason Avinger
+Jason Lee
+Jason Waters
+Jeremy Biddle
```
Comment (beeduul) [inline ACKNOWLEDGEMENTS.md]: ([@beeduul](https://github.com/beeduul))

**Comment by lanther** [comment]
Merging now - for further updates, please make a PR!

**Pull request merged**